# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/policy_low_level.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/policy_low_level.h
--- a/security/sandbox/chromium/sandbox/win/src/policy_low_level.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/policy_low_level.h	Wed Jun 12 11:10:48 2019 +0100
@@ -44,8 +44,7 @@
 
 // TODO(cpu): Move this constant to crosscall_client.h.
 const size_t kMaxServiceCount = 64;
-static_assert(IPC_LAST_TAG <= kMaxServiceCount,
-              "kMaxServiceCount is too low");
+static_assert(IPC_LAST_TAG <= kMaxServiceCount, "kMaxServiceCount is too low");
 
 // Defines the memory layout of the policy. This memory is filled by
 // LowLevelPolicy object.
@@ -120,7 +119,7 @@
 enum RuleOp {
   EQUAL,
   AND,
-  RANGE   // TODO(cpu): Implement this option.
+  RANGE  // TODO(cpu): Implement this option.
 };
 
 // Provides the means to collect a set of comparisons into a single
@@ -155,9 +154,7 @@
                       RuleOp comparison_op);
 
   // Returns the number of opcodes generated so far.
-  size_t GetOpcodeCount() const {
-    return buffer_->opcode_count;
-  }
+  size_t GetOpcodeCount() const { return buffer_->opcode_count; }
 
   // Called when there is no more comparisons to add. Internally it generates
   // the last opcode (the action opcode). Returns false if this operation fails.
@@ -180,8 +177,10 @@
   // addresses from opcode_start and copy the extra data (strings usually) into
   // decreasing addresses from data_start. Extra data is only present in the
   // string evaluation opcodes.
-  bool RebindCopy(PolicyOpcode* opcode_start, size_t opcode_size,
-                  char* data_start, size_t* data_size) const;
+  bool RebindCopy(PolicyOpcode* opcode_start,
+                  size_t opcode_size,
+                  char* data_start,
+                  size_t* data_size) const;
   PolicyBuffer* buffer_;
   OpcodeFactory* opcode_factory_;
   EvalResult action_;